src/modules/colorPicker/ColorPickerUI/Behaviors/GridEffectBehavior.cs

```diff
@@ -10,11 +10,14 @@ namespace ColorPicker.Behaviors
 {
     public class GridEffectBehavior : Behavior<FrameworkElement>
     {
-        private static double baseZoomImageSizeInPx = 50;
+        private static double _baseZoomImageSizeInPx = 50;
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1401:Fields should be private", Justification = "https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/dependency-property-security#:~:text=Dependency%20properties%20should%20generally%20be%20considered%20to%20be,make%20security%20guarantees%20about%20a%20dependency%20property%20value.")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2211:Non-constant fields should not be visible", Justification = "https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/dependency-property-security#:~:text=Dependency%20properties%20should%20generally%20be%20considered%20to%20be,make%20security%20guarantees%20about%20a%20dependency%20property%20value.")]
```

Comment:
@crutkas, Can't we just mark these Dependency Properties as readonly instead of suppressing these errors?

Reply:
I see why i did that.  https://github.com/microsoft/PowerToys/pull/5336#pullrequestreview-458962059 where i had them private as well.